spec/avram/type_extensions/bytes_spec.cr

```diff
@@ -0,0 +1,10 @@
+require "../../spec_helper"
+
+describe "Bytes" do
+  it "parses Bytes from String" do
+    result = Bytes.adapter.parse("10")
+    result.value.should eq(10)
```

Comment:
Looking at the FLoat64 and Int32 specs, should not the expected result be `"10".to_slice` instead of the integer 10?

Disconsider if the comment doesn't make sense since I didn't dig into Avram internals yet

Reply:
Oh, I just copied the spec. Since it doesn't compile yet, I haven't ran it, so I have no idea what to actually expect here. I've never used Bytes for anything (or a bytea column), so I'm not sure what a `String` -> `Bytes` conversion looks like :sweat_smile: